### ISSUE TITLE
Add a callback option

### ISSUE THREAD

**Issue by nilssolanki** [zengabor/zenscroll#12]
I'm doing some animations that depend on the scroll finishing. `setTimeout` is not accurate enough unfortunately.

Could you add an option to provide a callback on scrollend?

E.g. something like:

```
zenscroll.toY(50).onEnd(myCallback);
```

Promise-based would of course be even more beautiful, but it might bloat your library.

**Comment by zengabor** [comment]
That's a great idea. I could also use that. Thanks!

You are right, using promises would be very elegant but older browsers don't support it, and polyfills are quite large. I will probably just add another parameter at the end of each function, which is quite straightforward. It would look like this:

```
zenscroll.toY(50, null, myCallback)
```

(Where passing a falsy value like `null` as the second parameter makes sure that the default duration is used.)

**Comment by zengabor** [comment]
This is now implemented in version 3.1.0
Thanks for the great idea, @nilssolanki!

**Comment by nilssolanki** [comment]
Thank you for implementing this so quickly! I was gone for the weekend, so this was a beautiful surprise :)
